src/runtime/runtime.cs

```diff
@@ -106,7 +106,7 @@ public class Runtime
         internal static object IsFinalizingLock = new object();
         internal static bool IsFinalizing;
 
-        internal static bool Is32Bit = IntPtr.Size == 4;
+        internal static bool Is32Bit => IntPtr.Size == 4;
```

Comment:
Its' better to change it into readonly field, using property may make it check the condition everytimes.

Reply:
https://gist.github.com/lostmsu/ed31ec13815e2af674abd96d0a4fc8ae
|         Method |           Job |       Runtime |      Mean |    Error |   StdDev |
|--------------- |-------------- |-------------- |----------:|---------:|---------:|
| StaticReadonly |      .NET 4.8 |      .NET 4.8 |  93.69 us | 1.812 us | 2.541 us |
|         Inline |      .NET 4.8 |      .NET 4.8 |  92.48 us | 1.498 us | 1.402 us |
| StaticReadonly | .NET Core 3.1 | .NET Core 3.1 | 188.60 us | 3.608 us | 4.155 us |
|         Inline | .NET Core 3.1 | .NET Core 3.1 |  94.08 us | 1.868 us | 1.919 us |
| StaticReadonly |          Mono |          Mono |  92.82 us | 1.824 us | 2.172 us |
|         Inline |          Mono |          Mono |  93.29 us | 1.831 us | 2.035 us |

Reply:
https://gist.github.com/amos402/7bd6adab1fe489c0552a109a8b3dff28
Haven't been use that benchmark framework before so I wrote it by raw.
Forget the errors, seems they were same after optimized. Haven't dig in their JITed code, I'm not sure either the usage striped by optimization or just generated the same machine code.
Never mind, it won't be big deal always.

Debug:
```
.NET Framework 4.8.4150.0
is32bit: False
Is32Bit: False
StaticReadonly best: 00:00:00.1345072
Inline best: 00:00:00.2713410
Press any key to continue . . .
```

Release:
```
.NET Framework 4.8.4150.0
is32bit: False
Is32Bit: False
StaticReadonly best: 00:00:00.0660235
Inline best: 00:00:00.0660266
Press any key to continue . . .
```